.git-blame-ignore-revs

```diff
@@ -0,0 +1 @@
+c178e0b482d65919a4c9b63e7b0e03974ec9b9f1
```

Comment:
Will this still work for `main` if the PR is squash-merged?

Tangent for a possible separate PR: can commits be added retrospectively to this file?  I’m thinking of the various `black` updates.

Reply:
Good point, in this case we could choose to 

1. not squash this PR
2. squash it and follow-up with a single commit pull-request with the sha of the previous squashed pull-request commit
3. ditch the use of `.git-blame-ignore-revs` as a concept

Reply:
@rcomer Yup, okay, I'll hunt those commits down and look to add them.

I'm in total agreement with you, I'm interested in the code changes, not so much style.

Reply:
Might also be good to add comment so future us knows why each commit was skipped.
https://github.com/matplotlib/matplotlib/blob/main/.git-blame-ignore-revs

Reply:
Done. See [08cb0a5](https://github.com/SciTools/iris/pull/5186/commits/08cb0a5d4931026385579d889b1283d1c1a633df)